src/authentication.ts

```diff
@@ -69,30 +81,30 @@ export const fetchTokens = (config: TInternalConfig): Promise<TTokenResponse> =>
     throw Error("Can't get tokens without the CodeVerifier. \nHas authentication taken place?")
   }
 
-  const formData = new FormData()
-  formData.append('grant_type', 'authorization_code')
-  formData.append('code', authCode)
-  formData.append('scope', config.scope)
-  formData.append('client_id', config.clientId)
-  formData.append('redirect_uri', config.redirectUri)
-  formData.append('code_verifier', codeVerifier)
-
-  return postWithFormData(config.tokenEndpoint, formData)
+  const tokenRequest: TTokenRequest = {
```

Comment:
```suggestion
  const tokenRequest: TTokenRequestWithCodeAndVerifier = {
```
Since we now have these explicit types, I think it makes sense to use them here.

Reply:
Yes, agree.